blake3/main.rs

```diff
@@ -1,7 +1,7 @@
 fn main() -> std::io::Result<()> {
     let data: Vec<u8> = random_numbers()
         .flat_map(|it| it.to_le_bytes())
-        .take(64 * 1024)
```

Comment:
Yeah, 64k is a somewhat intentional number here: at TigerBeetle, that’s the block size (IO happens at the block granularity)

https://github.com/tigerbeetledb/tigerbeetle/blob/46c00e284b7be627fb90a23a0c2d138779a2fadc/src/config.zig#L95

Still an interesting bench, just for a slightly different use-case, merging!